src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -96,4 +98,70 @@ static String formatedValue(double rms) {
     public List<VCFInfoHeaderLine> getDescriptions() {
         return Arrays.asList(VCFStandardHeaderLines.getInfoLine(getKeyNames().get(0)), GATKVCFHeaderLines.getInfoLine(getRawKeyName()));
     }
+
+    /**
+     * converts {@link GATKVCFConstants#RAW_RMS_MAPPING_QUALITY_KEY} into  {@link VCFConstants#RMS_MAPPING_QUALITY_KEY}  annotation if present
+     * @param vc which potentially contains rawMQ
+     * @return if vc contained {@link GATKVCFConstants#RAW_RMS_MAPPING_QUALITY_KEY} it will be replaced with {@link VCFConstants#RMS_MAPPING_QUALITY_KEY}
+     * otherwise return the original vc
+     */
+    public VariantContext finalizeRawMQ(final VariantContext vc) {
+        final String rawMQdata = vc.getAttributeAsString(getRawKeyName(), null);
+        if (rawMQdata == null) {
+            return vc;
+        } else {
+            final double squareSum = parseRawDataString(rawMQdata);
+            final String finalizedRMSMAppingQuality = makeFinalizedAnnotationString(vc, squareSum);
+            final VariantContext result = new VariantContextBuilder(vc)
+                    .rmAttribute(getRawKeyName())
+                    .attribute(getKeyNames().get(0), finalizedRMSMAppingQuality)
+                    .make();
+            return result;
+        }
+    }
+
+    private static double parseRawDataString(String rawDataString) {
+        try {
+            final double squareSum = Double.parseDouble(rawDataString.split(",")[0]);
+            return squareSum;
+        } catch (final NumberFormatException e){
+            throw new UserException.BadInput("malformed " + GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY +" annotation: " + rawDataString);
+        }
+
+    }
+
+
+    private static String makeFinalizedAnnotationString(final VariantContext vc, final double squareSum) {
+        final int numOfReads = getNumOfReads(vc);
+        return formatedValue(Math.sqrt(squareSum /numOfReads));
```

Comment:
Explicitly cast `numOfReads` to `double` to force floating-point division even if the type of the `squareSum` argument were ever to change.